components/scream/src/physics/share/tests/physics_test_data_unit_tests.cpp

```diff
@@ -35,6 +35,20 @@ struct FakeClass1 : public PhysicsTestData
   PTD_STD_DEF(FakeClass1, 5, dim1, dim2, dim3, dim4, scalar);
 };
 
+struct AllTrue : public PhysicsTestData
```

Comment:
Is there already a use case for this class? Meaning, some code that needs this?

Reply:
@bartgol , yes, @xyuan asked me for this.